tests/io/write_mesh_particles.cc

```diff
@@ -110,6 +112,23 @@ bool write_json(unsigned dim, bool resume, const std::string& analysis,
   return true;
 }
 
+// Write JSON Entity Set
+bool write_entity_set() {
+  // JSON Entity Sets
+  Json json_file = {
+      {"particle_sets",
+       {{{"id", 2},
+         {"set", {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}}}}}};
+
+  // Dump JSON as an input file to be read
+  std::ofstream file;
+  file.open("entity_sets_0.json");
+  file << json_file.dump(2);
+  file.close();
+
+  return true;
```

Comment:
Since this is a `bool`, what is the case when it gives `false`?

Reply:
It doesn't seem like any of the functions within `write_mesh_particles` ever return `false` so I matched the existing structure

Reply:
Alright, I am OK with this!